src/main/java/uk/gov/justice/digital/delius/data/api/OffenderRelease.java

```diff
@@ -15,8 +15,6 @@
 public class OffenderRelease {
     @ApiModelProperty(value = "The date the release occurred", example = "2019-11-26")
     private LocalDate date;
-    @ApiModelProperty(value = "The reason for the release")
```

Comment:
Would ReleaseType not be a the release reason? Maybe look at the values or change the model to be `releaseType` rather than `reason`